com.unity.render-pipelines.universal/Runtime/Passes/CopyDepthPass.cs

```diff
@@ -103,13 +103,13 @@ public override void Execute(ScriptableRenderContext context, ref RenderingData
             // Currently the y-flip is handled in CopyDepthPass.hlsl by checking _ProjectionParams.x
             // If you replace this Blit with a Draw* that sets projection matrix double check
             // to also update shader.
-            // scaleBias.x = flipSign
-            // scaleBias.y = scale
-            // scaleBias.z = bias
-            // scaleBias.w = unused
+            // scaleBiasDstUV.x = flipSign
+            // scaleBiasDstUV.y = scale
+            // scaleBiasDstUV.z = bias
+            // scaleBiasDstUV.w = unused
             float flipSign = (cameraData.IsCameraProjectionMatrixFlipped()) ? -1.0f : 1.0f;
-            Vector4 scaleBiasRt = (flipSign < 0.0f) ? new Vector4(flipSign, 1.0f, -1.0f, 1.0f) : new Vector4(flipSign, 0.0f, 1.0f, 1.0f);
-            cmd.SetGlobalVector(ShaderPropertyId.scaleBiasRt, scaleBiasRt);
+            Vector4 scaleBiasDstUV = (flipSign < 0.0f) ? new Vector4(flipSign, 1.0f, -1.0f, 1.0f) : new Vector4(flipSign, 0.0f, 1.0f, 1.0f);
```

Comment:
maybe we should have a helper method for this? What is the workflow for users that want to do blit in URP? F.ex, now the 2nd most voted card and feedback is about custom post-processing.

Seems like we need a Blit/DrawFullscreen helper here to encapsulate this scale bias logic.

Reply:
Thanks felipe. I agree we should use helper function. I will look into this.